src/ensembl/src/content/app/browser/browserActions.ts

```diff
@@ -149,15 +135,14 @@ export const updateTrackStatesAndSave: ActionCreator<
 export const toggleBrowserNav = createAction(
   'browser/toggle-browser-navigation',
   (resolve) => {
-    return () => resolve(undefined, getBrowserAnalyticsObject('Navigation'));
+    return () => resolve(undefined);
```

Comment:
Since nothing is passed to `resolve` can this action be created using `createStandardAction`?

Reply:
That's done.